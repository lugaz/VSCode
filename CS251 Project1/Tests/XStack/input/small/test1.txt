peek
push 514
push 37
push 431
pop
peek
peek
pop
pop
push 491
push 966
push 911
