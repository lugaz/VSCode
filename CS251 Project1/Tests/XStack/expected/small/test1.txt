-1
431
37
37
37
514
